parsexml.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+#Author:        Sasan Bahadaran
+#Date:          2/29/16
+#Organization:  Commerce Data Service
+#Description:   This script crawls the files directory and finds the metadata
+#xml file for each set of pdf files.  It then gets the document id from
+#the metadata and finds the associated extracted pdf text and adds it to the
+#structure of the document, then transforms the document to json and saves it
+#to a file.
+
+import json, xmltodict, os, logging, time
+
+#get value for specified tag from a list
+def getData(list,tag):
```

Comment:
Not sure why `getData` or `setData` are needed here, rather than just accessing the dict directly. On the `get` side, there's already the `.get()` method, which will not throw an error if the key does not exist.

    x.get('DOCUMENT_IMAGE_ID')